src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -697,6 +697,8 @@ public void TestProcessStartTime()
                 p.StartInfo.RedirectStandardInput = true;
                 p.Start();
                 Assert.Equal(p.StartTime, p.StartTime);
+                var p2 = Process.GetProcessById(p.Id);
+                Assert.Equal(p.StartTime, p2.StartTime);
```

Comment:
This test is failed on OSX. Seems like I should implement this specific test specially for Linux. What do you think?

Reply:
Moved to ProcessTests.Linux